README.md

```diff
@@ -1 +1,72 @@
-# ds-helper
\ No newline at end of file
+---
+__Let's Learn Some Basic GitHub Steps :)__
+
+- __[KOMEN.CO](http://www.estudiosdemercado.org/komenco.jpg)__ -The Beginning of your Journey
+![Minion](http://www.estudiosdemercado.org/komenco.jpg)
+
+__Following are basic GitHub Steps!__
+
+---
+## Steps:
+`code`
+
+ 0 step: Switch to the local main branch
```

Comment:
Lets use a numbered list. Please refer to https://stackoverflow.com/questions/18088955/markdown-continue-numbered-list

Also, you can start indexing from '1' instead of '0'